### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by wlhlm**

### CODE REVIEW

File: resources/assets/minefactoryreloaded/language/de_DE.lang [hunk 1]
```diff
@@ -157,9 +163,16 @@ tile.mfr.cable.redstone.name = RedNet-Kabel
 tile.mfr.cable.redstone.glass.name = RedNet-Kabel
 tile.mfr.cable.redstone.energy.name = RedNet-Energiekabel
 tile.mfr.cable.redstone.energyglass.name = RedNet-Energiekabel
+tile.mfr.cable.plastic.name = Plastikrohr
+
 tile.mfr.rednet.logic.name = Programmierbarer RedNet-Controller
 tile.mfr.rednet.panel.historian.name = Graph
 
+tile.mfr.machineblock = item.mfr.machineblock
+tile.mfr.machineblock.prc.name = Programmierbarer RedNet-Controller-Gehäuse
```
Comment (Vexatos) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: How about "Gehäuse für den Programmierbaren RedNet-Controller"?

File: resources/assets/minefactoryreloaded/language/de_DE.lang [hunk 2]
```diff
@@ -209,31 +218,57 @@ item.mfr.syringe.growth.name = Wachstumsspritze
...
+item.mfr.fertilizer.name = Industriedünger
 item.mfr.milkbottle.name = Milchflasche
 item.mfr.spyglass.name = Fernglas
+item.mfr.ruler.name = Lineal
 item.mfr.xpextractor.name = Erfahrungsextrakteur
 item.mfr.portaspawner.name = Portalspawner
 item.mfr.straw.name = Strohhalm
+
 item.mfr.rednet.meter.name = RedNet-Messgerät
+item.mfr.rednet.meter.info.name = RedNet Multimeter
```
Comment (Vexatos) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: Maybe a hyphen here?

File: resources/assets/minefactoryreloaded/language/de_DE.lang [hunk 3]
```diff
@@ -368,6 +404,10 @@ circuit.mfr.xor.4 = XOR (4 Eingänge)
 container.mfr.radius = Radius:
 container.mfr.power = Energieverbrauch:
 
+container.mfr.autobrewer.resources = Resourcen
```
Comment (Vexatos) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: Needs to be "Ressourcen".

File: resources/assets/minefactoryreloaded/language/de_DE.lang [hunk 4]
```diff
@@ -401,36 +442,55 @@ chat.info.mfr.rednet.connection.forced = Seite auf Erzwungenen Verbindungsmodus
...
+# Tooltips: these can be added to any item by a lang file, but they cannot be removed
+# this is a limitation of the localization system.
+# Replace "item." or "tile." with "tip.info" and remove ".name" and that tooltip will be
+# displayed under the item with exceptions, such as the PRC which uses a custom entry for legacy reasons
+ 
+tip.info.mfr.hammer = Wie eine Zange, nur mehr zertrümmernd
+ 
+tip.info.mfr.rubberwood.sapling.massive = Warnung: Packratt's Weltenschiff.
+ 
+tip.info.mfr.prc = Wenn du nicht mehr weißt, wies weitergeht, lese die Anleitung.
```
Comment (Vexatos) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: "wie es weitergeht".

**Comment by Vexatos** [comment]
Other than that, not bad at all! Thanks for updating!

File: resources/assets/minefactoryreloaded/language/de_DE.lang (hunk 1 above)
Comment (wlhlm) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: Hmm, not sure. I think adding the "Gehäuse"-suffix brings it more in line with the translation above ([Line 168](https://github.com/wlhlm/MineFactoryReloaded/blob/update-german-translation/resources/assets/minefactoryreloaded/language/de_DE.lang#L168)):
```
tile.mfr.rednet.logic.name = Programmierbarer RedNet-Controller
```
So you'll have "\[Controller]" and "\[Controller] - Gehäuse".

(I'd add spaces around the hyphen to break up the last word)

File: resources/assets/minefactoryreloaded/language/de_DE.lang (hunk 2 above)
Comment (wlhlm) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: Will do.

File: resources/assets/minefactoryreloaded/language/de_DE.lang (hunk 3 above)
Comment (wlhlm) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: :+1:

File: resources/assets/minefactoryreloaded/language/de_DE.lang (hunk 4 above)
Comment (wlhlm) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: :+1:

File: resources/assets/minefactoryreloaded/language/de_DE.lang (hunk 1 above)
Comment (Vexatos) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: Well but the way you currently have it makes the "Programmierbarer" be grammatically incorrect, while there would be no way to make it correct. The "Gehäuse für" is the way other mods like Forestry handle such a strange case.

Comment (wlhlm) [inline resources/assets/minefactoryreloaded/language/de_DE.lang]: Didn't know about the Forestry thing. Will change accordingly.

**Comment by wlhlm** [comment]
Updated the branch with all suggestions.

Thanks for the feedback!

**Pull request merged**